components/DataLiberation/Tests/css-test-cases.php

```diff
@@ -0,0 +1,5390 @@
+<?php
+/**
+ * CSS Tokenizer Test Cases generated from @rmenke/css-tokenizer-tests.
+ * Indices are UTF-8 byte offsets.
+ * String and URL values are already decoded.
+ */
+
+return array (
+  'tests/at-keyword/0001' => 
+  array (
+    'css' => '@foo
+',
```

Comment:
is this auto-generated? If so, I have found it useful for the sake of readability to replace literal whitespace with Unicode escapes, making them ever-more explicit as being text under test.

e.g. `'css' => "@foo\n"`

Reply:
Yes, this was auto-generated with a script – these are exactly the line endings I've meant in my last comment. I've updated it to use escape sequences (`\n`) instead of literal codepoints (newlines, carriage returns etc).